src/Artsy/Router/__tests__/buildServerApp.test.tsx

```diff
@@ -4,21 +4,19 @@
 
 import { ContextConsumer } from "Artsy"
 import { createRelaySSREnvironment } from "Artsy/Relay/createRelaySSREnvironment"
-import { buildServerApp, ServerRouterConfig } from "Artsy/Router/buildServerApp"
+import {
+  __THOU_SHALT_NOT_FAFF_AROUND_WITH_THIS_HERE_OBJECT_WE_ARE_SERIOUS__,
```

Comment:
⚔️

Reply:
It felt time that we had one of our one! 🧐